.github/workflows/tasklist-ci.yml

```diff
@@ -35,6 +35,31 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  check_changes:
+    name: Check for changes in the client folder
+    runs-on: ubuntu-latest
+    outputs:
+      has_changed_frontend: ${{ steps.filter.outputs.src_changed }}
+    steps:
+      - uses: actions/checkout@1d96c772d19495a3b5c517cd2bc0cb401ea0529f # v4
+        with:
+          fetch-depth: 0
+      - id: filter
+        run: |
+          echo "src_changed=false" >> $GITHUB_OUTPUT
+          if [ "${{ github.event_name }}" = "pull_request" ]; then
+            # For pull requests, compare the PR base and head commits
+            git fetch origin ${{ github.base_ref }}
+
+            if git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} | grep -q 'tasklist/client/'; then
+              echo 'src_changed=true' >> $GITHUB_OUTPUT
+            fi
+          elif [ "${{ github.event_name }}" = "push" ]; then
+            # For pushes, compare the current commit with the previous one
+            if git diff --name-only ${{ github.event.before }} ${{ github.event.after }} | grep -q 'tasklist/client/'; then
+              echo 'src_changed=true' >> $GITHUB_OUTPUT
+            fi
+          fi
```

Comment:
If any of you have an idea to replace this with something simpler I'm open to it.
We could also create a Tasklist FE CI workflow and then we can use the trigger path filters to achieve the same